### PR TITLE
ContentExchange Bid Adapter: add new bid adapter

### PR DESCRIPTION
**Pull request by contentexchange**
## Type of change

- [x] New bidder adapter   

## Description of change

ContentExchange bidder adapter

- test parameters for validating bids
```
var adUnits = [
  {
      code: 'adunit1',
      mediaTypes: {
          banner: {
              sizes: [ [300, 250], [320, 50] ],
          }
      },
      bids: [
          {
              bidder: 'contentexchange',
              params: {
                  placementId: '0',
                  adFormat: 'banner'
              }
          }
      ]
  },
  {
      code: 'addunit2',
      mediaTypes: {
          video: {
              playerSize: [ [640, 480] ],
              context: 'instream',
              minduration: 5,
              maxduration: 60,
          }
      },
      bids: [
          {
              bidder: 'contentexchange',
              params: {
                  placementId: '0',
                  adFormat: 'video'
              }
          }
      ]
  },
  {
      code: 'addunit3',
      mediaTypes: {
          native: {
              title: {
                  required: true
              },
              body: {
                  required: true
              },
              icon: {
                  required: true,
                  size: [64, 64]
              }
          }
      },
      bids: [
          {
              bidder: 'contentexchange',
              params: {
                  placementId: '0',
                  adFormat: 'native'
              }
          }
      ]
  }
];
```

- contact email of the adapter’s maintainer: no-reply@vsn.si
- [x] official adapter submission
- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/pull/3146

### CODE REVIEW

**Comment by contentexchange** [comment]
@Rothalack, please pay attention to this pull request

**Comment by contentexchange** [comment]
@ChrisHuie, please pay attention to this pull request

**Comment by Rothalack** [comment]
@contentexchange Apologies, I had written these almost two weeks ago now but apparently never submitted them to where you could see them.

File: test/spec/modules/contentexchangeBidAdapter_spec.js [hunk 1]
```diff
@@ -0,0 +1,399 @@
...
+        }
+      },
+      params: {
+        placementId: 'test',
+        adFormat: BANNER
+      }
+    },
+    {
+      bidId: getUniqueIdentifierStr(),
+      bidder,
```
[the remark this answers is not in the document]
Comment (contentexchange) [reply test/spec/modules/contentexchangeBidAdapter_spec.js]: Ok, i fixed this moment

File: modules/contentexchangeBidAdapter.js [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+  const placement = {
+    placementId,
+    bidId,
+    adFormat,
+    schain,
+    bidfloor
+  };
+
+  switch (adFormat) {
+    case BANNER:
```
[the remark this answers is not in the document]
Comment (contentexchange) [reply modules/contentexchangeBidAdapter.js:43]: Sorry, but I didn't understand what was wrong with this place. I get BANNER, VIDEO and NATIVE constants from `src/mediaTypes.js` where they meant 'banner', 'video' and 'native', respectively.

[the remark this answers is not in the document]
Comment (Rothalack) [reply modules/contentexchangeBidAdapter.js:43]: That makes sense.

**Review by Rothalack — APPROVED** [review]

**Pull request merged**